Homework_6/Calculator.java

```diff
@@ -0,0 +1,42 @@
+package Homework_6;
+
+public class Calculator {
+    public static int add(int a, int b) {
+    return a + b;
```

Comment:
Tab here

Reply:
return statement should not be at the same level as public.
Please, do it like in subtract method